modules/eks/ray-orchestrator/modulestack.yaml

```diff
@@ -0,0 +1,84 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This stack deploys a Module specific IAM permissions
+
+Parameters:
+  ProjectName:
+    Type: String
+    Description: The name of the Project
+  DeploymentName:
+    Type: String
+    Description: The name of the deployment
+  ModuleName:
+    Type: String
+    Description: The name of the Module
+  RoleName:
+    Type: String
+    Description: The name of the IAM Role
+  EksClusterAdminRoleArn:
+    Type: String
+    Description: Role from the EKS module with cluster admin permissions
+
+Resources:
+  Policy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Action:
+              - "sts:GetCallerIdentity"
+            Effect: Allow
+            Resource:
+              - "*"
+          - Action:
+              - "iam:CreatePolicy"
+              - "iam:DetachRolePolicy"
+              - "iam:DeletePolicy"
+              - "iam:DeleteRolePolicy"
+              - "iam:CreateRole"
+              - "iam:DeleteRole"
+              - "iam:AttachRolePolicy"
+              - "iam:PutRolePolicy"
+              - "iam:UpdateRole"
+              - "iam:CreatePolicyVersion"
+              - "iam:DeletePolicyVersion"
+              - "iam:TagRole"
+              - "iam:UntagRole"
+              - "iam:PassRole"
+            Effect: Allow
+            Resource:
+              - !Sub "arn:aws:iam::${AWS::AccountId}:role/${ProjectName}-${DeploymentName}-${ModuleName}*"
+              - !Sub "arn:aws:iam::${AWS::AccountId}:role/service-role/${ProjectName}-${DeploymentName}-${ModuleName}*"
```

Comment:
Not sure if it matters, but these should probably be partition agnostic?

Reply:
It does - thanks. Updated.